### PR TITLE
Added PixiParser.Skia package and remove System.Drawing.Common dependency

### PR DESCRIPTION
**Pull request by CPKreuz**
**Groups do not work yet**

Version 1.2.1 removes the need for the System.Drawing.Common package, which is only available for Windows

It also adds a new package called PixiEditor.PixiParser.Skia which contains extension methods for creating Skia Bitmaps from Layers or using Skia Bitmaps to fill the PNG data of a layer

Last change is that it's now targeting .NET 5 and .NET Standard 2.0 (previously just .NET Standard 2.1)

### CODE REVIEW

**Review by flabbet — COMMENTED** [review]
Does it contain backwards compatibility?

**Comment by flabbet** [comment]
Also, I am not going to approve, because of not supporting groups

**Comment by CPKreuz** [comment]
> Does it contain backwards compatibility?

Only if the document does not contain any groups

File: src/PixiParser/Helpers/StructureHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                {
+                    group.SerializedStartLayer++;
+                }
+
+                if (group.SerializedEndLayer >= start)
+                {
+                    group.SerializedStartLayer++;
+                }
+            }
+        }
```
Comment (flabbet) [inline src/PixiParser/Helpers/StructureHelpers.cs:30]: What is this used for?

Comment (flabbet) [reply src/PixiParser/Helpers/StructureHelpers.cs:30]: Ok nvm, I understood it

**Review by flabbet — APPROVED** [review]

**Pull request merged**